File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.go
func longestConsecutive(nums []int) int {

    max := 1
    hash := make(map[int]int)
    if len(nums) == 0 {
        return 0
    }
    for _,val := range nums {
        hash[val] = 1
    }

    for key,_ := range hash {
        _,pre := hash[key-1]
        if pre {
            continue
        }
        cur:= key
        curMax := 1
         _,find := hash[cur]
        for ;find;cur++ {
             _,find = hash[cur]
            curMax += 1
        }
        if curMax > max {
            max = curMax
        }
    }
    return max -2
    
}

File: Data Structures & Algorithms/maximum-subarray/submission-1.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        cur = -1
        maxval = nums[0]
        for val in nums:
            if cur < 0:
                cur = 0
            cur += val
            maxval = max(maxval , cur)
        
        return maxval